xtask/src/commands/release.rs

```diff
@@ -15,6 +23,21 @@ pub use tag_releases::*;
 
 #[derive(Debug, Subcommand)]
 pub enum Release {
+    /// Create a release plan. This is the first step in the release process.
+    /// Accepts zero or more package names. If no package names are
+    /// specified, all packages are included.
+    ///
+    /// The result of this command is a json file that can be customized to
+    /// control what and how gets released.
+    #[cfg(feature = "release")]
+    Plan(PlanArgs),
+    /// Apply a release plan.
+    ///
+    /// This command takes a release plan, makes changes to the codebase based
+    /// on that plan, and commits the result on a new branch. The command will
+    /// then open a pull request for the branch.
+    #[cfg(feature = "release")]
+    ApplyPlan(ApplyPlanArgs),
```

Comment:
Nit/Bikeshed: thoughts on `ExecutePlan`? Then we'd have `Release: Plan -> Execute -> Publish`.

Reply:
My only concern is that "execute" implies something final, but we're only around halfway through a release process here. Still, might be a good name.